paddle/fluid/operators/fill_constant_op.cc

```diff
@@ -70,6 +70,12 @@ class FillConstantOp : public framework::OperatorBase {
   }
 };
 
+class FillConstantOpVarTypeInference : public framework::VarTypeInference {
+ public:
+  void operator()(const framework::OpDesc &op_desc,
+                  framework::BlockDesc *block) const override {}
```

Comment:
Why declare an empty one?

Reply:
If ```FillConstantOp``` do not have a ```FillConstantOpVarTypeInference ``` class, the default 
```VarTypeInference ``` method will set all vars' type to ```LodTensor```, It will change ```SelectedRow``` to ```LodTensor```.

Reply:
@chengduoZH  seems the default one is not good...